src/runtime/server/server.ts

```diff
@@ -106,9 +108,47 @@ export function create(appState: AppState) {
           loadedRuntimePlugins
         )
 
+        /**
+         * Resolve if subscriptions are enabled or not
+         */
+
+        let enableSubscriptionsServer: boolean = false
+
+        if (settings.metadata.fields.subscriptions.fields.enabled.from === 'change') {
```

Comment:
There is a bug in `Setset` preventing this from working right now. At least statically.